applications/vessel-history/src/data/config.ts

```diff
@@ -17,7 +21,32 @@ export const BACKGROUND_LAYER = [
     color: MAP_BACKGROUND_COLOR,
   } as BackgroundGeneratorConfig,
 ]
-export const DEFAULT_DATAVIEWS = [
+export const DEFAULT_DATAVIEWS: GeneratorConfig[] = [
+  // When offline we serve this landmass layer as a backup
+  // in case that not all tiles are cached
+  {
+    id: 'landmass_offline',
+    type: Generators.Type.GL,
+    sources: [
+      {
+        type: 'geojson',
+        data: '/data/ne_10m_admin_0_countries_gj.geojson',
+      },
+    ],
+    layers: [
+      {
+        type: 'fill',
+        paint: {
+          'fill-color': DEFAULT_LANDMASS_COLOR,
+          'fill-opacity': 0.99, // This is a trick to allow proper rendering of MGL heatmap layers behind
+        },
+      } as FillLayer,
+    ],
+    metadata: {
+      offline: true,
```

Comment:
There's no switch yet. Given that this layer is already cached by the sw it's rendered behind the landmass tiles, so that when the landmass tiles (high res) are not available this layer displays the landmass

Reply:
I see! it is always there and the cache is done by default! Thanks for the clarification.